pkg/api/v1alpha1/cluster_webhook_test.go

```diff
@@ -839,7 +839,7 @@ func TestClusterValidateUpdateIdentityProviderRefsImmutable(t *testing.T) {
 	}
 
 	g := NewWithT(t)
-	g.Expect(c.ValidateUpdate(cOld)).NotTo(Succeed())
+	g.Expect(c.ValidateUpdate(cOld)).To(Succeed())
```

Comment:
all the tests you changed need to be renamed to "Mutable" e.g.`TestClusterValidateUpdateIdentityProviderRefsMutablexxx`